src/views/Home/Home.test.jsx

```diff
@@ -1,15 +1,34 @@
+import { render, screen } from '@testing-library/react'
+import Profile from '../../components/Profile/Profile'
 
-const user = {
-  id: 1,
-  created_at: '2021-12-13T00:17:29+00:00',
-  name: 'Vonta',
-  avatar: 'https://thumbs.gfycat.com/NiceRequiredGrunion-size_restricted.gif',
-  header: 'https://static.wikia.nocookie.net/naruto/images/5/50/Team_Kakashi.png',
-  likes: ['React', 'Anime', 'Traveling', 'Living', 'Tower Defense Games', 'Card Games'],
-  motto: 'Res Non Verba',
-  color: 'crimson',
-}
+describe('App', () => {
+  const user = {
+    id: 1,
+    created_at: '2021-12-13T00:17:29+00:00',
+    name: 'Vonta',
+    avatar: 'https://thumbs.gfycat.com/NiceRequiredGrunion-size_restricted.gif',
+    header: 'https://static.wikia.nocookie.net/naruto/images/5/50/Team_Kakashi.png',
+    likes: ['React', 'Anime', 'Traveling', 'Living', 'Tower Defense Games', 'Card Games'],
+    motto: 'Res Non Verba',
+    color: 'crimson',
+  }
+  it('Should render the user profile information just as name, avatar, motto. etc..,', async () => {
+    render(<Profile user={user} />)
 
-test('Should render the user profile', () => {
+    const name = screen.getByText('Vonta')
+    const avatar = screen.getByAltText('avatar')
+    const header = screen.getByAltText('header')
+    const motto = await screen.findByLabelText('motto')
+    const likes = screen.queryByText('Anime')
```

Comment:
Here we will want to do a getByText('Interests') to test for the interests heading. We will also want to test for the list of user likes, such as the length of the likes, using a findByRole('list').